exercises/numbers/remainderOf/remainderOf.js

```diff
@@ -15,17 +15,25 @@
  * @param {number} num The input number
  * @returns {boolean} True if num is even and false otherwise
  */
+
 function remainderOf(n, d) {
   /*
     This is your job. :)
 
     If you're not sure, step out of the code and use pen + paper. Start
     with simple examples and pay attention to the process you carry out.
   */
+return n%d
 }
 
 if (require.main === module) {
   console.log('Running sanity checks for remainderOf');
+  console.log(remainderOf(10,1));
+  console.log(remainderOf(10, 1)); // => 0
+  console.log(remainderOf(10, 2)); // => 0
+  console.log(remainderOf(10, 3)); // => 1
```

Comment:
Consider writing something like:

```js
console.log(remainderOf(10, 3) === 1);
```

This way you can see immediately in the terminal whether your code is doing what you expect. If you see `false` anywhere you know something is wrong. By printing out the actual values, you have to remember what each line is supposed to be.

Reply:
Thanks!